portmap/core/track.py

```diff
@@ -0,0 +1,18 @@
+from functools import wraps
+
+from .models import TrackArticleView
+
+def track_view(function):
+    @wraps(function)
+    def _wrap_track_view(request, *args, **kwargs):
+        print("Tracking: " + request.path)
+
+        # Get event properties for tracking the view
+        article_path = request.path
+        allowed_attr = ['HTTP_ACCEPT','HTTP_ACCEPT_ENCODING','HTTP_HOST','HTTP_REFERER','HTTP_USER_AGENT','QUERY_STRING','REQUEST_METHOD']
+        user_meta_data = {key: request.META[key] for key in allowed_attr if key in request.META}
```

Comment:
We can't track all this; we don't want to track users this much and our privacy policy indicates we don't.  Rather than grab all the data we can, we need to figure out which data we want and what for.